packages/edit-widgets/src/components/widget-areas-block-editor-provider/index.js

```diff
@@ -99,22 +99,26 @@ export default function WidgetAreasBlockEditorProvider( {
 	);
 
 	return (
-		<ShortcutProvider>
-			<BlockEditorKeyboardShortcuts.Register />
-			<KeyboardShortcuts.Register />
-			<SlotFillProvider>
-				<ExperimentalBlockEditorProvider
-					value={ blocks }
-					onInput={ onInput }
-					onChange={ onChange }
-					settings={ settings }
-					useSubRegistry={ false }
-					{ ...props }
-				>
-					<CopyHandler>{ children }</CopyHandler>
-					<ReusableBlocksMenuItems rootClientId={ widgetAreaId } />
-				</ExperimentalBlockEditorProvider>
-			</SlotFillProvider>
-		</ShortcutProvider>
+		<StrictMode>
```

Comment:
Yes, that's a good catch, a forgotten legacy use of `render` :) I fixed it in f0fb47f7c0.

And I fixed it in a way that's IMO slightly better that how we do it in other apps. Instead of rendering `<StrictMode>` inside the top-level component, I use `<StrictMode>` as the very top-level component, in the `render` call.

Because otherwise, React devtools show a warning that the top level component is not in strict mode. Because only its children are:

<img width="304" alt="Screenshot 2023-02-02 at 10 03 48" src="https://user-images.githubusercontent.com/664258/216280082-2bd5e623-5fb1-44e0-86af-87b7ef56f77a.png">

A completely harmless warning, but still a warning.

Reply:
😅 I've rerun them just in case it was a flaky run. Feel free to revert the change and leave it for another PR if you feel it introduces extra flakiness, though

Reply:
I was able to figure out the failures and fix them.

One was a buggy adding and removing event listeners in `FocusControl`. The `useEffect` cleanup function removes listener that was never added from an object that doesn't exist, leading to crash. Caused by React unmounting the effect and then mounting it again.

Then there was a check if the editor is in navigation mode, which was searching for dynamic "you are currently in navigation mode" text element, added by `speak()`. But there are two `speak()` calls racing with each other, editor mode switch and block selection, and concurrent mode reversed their order. I fixed that by searching for `.is-navigate-mode` CSS selector instead.

The last one is a pre-existing failure where the element to click on is obcured by block toolbar:

<img width="699" alt="Screenshot 2023-02-02 at 13 39 09" src="https://user-images.githubusercontent.com/664258/216329089-b410b71b-6369-40da-9066-84c85a41c86b.png">

This has been auto-reported in #38832 for almost a year. I fixed this by first clicking on the page text, unfocusing the editor and hiding the toolbar, and only then clicking on the blue round "edit widget" icon that's no longer obscured.

Reply:
> This has been auto-reported in https://github.com/WordPress/gutenberg/issues/38832 for almost a year.

Seems like it's a different failure though. 🤔 
I think the reported issue might be a legit flaky behavior of the customizer but I didn't verify it 😅

Reply:
I saw this block toolbar overlap when running this locally. Also, the `failures-artifact` zip file will have screenshots when this fails in CI, can be inspected.